static/js/directives/audiovideo.js

```diff
@@ -244,8 +244,13 @@ define(['jquery', 'underscore', 'text!partials/audiovideo.html', 'text!partials/
 
 			$scope.toggleFullscreen = function() {
 				//console.log("Toggle full screen", BigScreen.enabled, $scope.isActive, $scope.hasUsermedia);
+				var target = $scope.layoutparent[0];
+				// Display whole page in fullscreen when in specific layout modes
+				if ($scope.layout.presentation || $scope.layout.youtubevideo) {
+					target = null;
+				}
 				if (BigScreen.enabled && ($scope.isActive || $scope.hasUsermedia)) {
-					BigScreen.toggle($scope.layoutparent[0], function() {
+					BigScreen.toggle(target, function() {
```

Comment:
When no target is passed [BigScreen uses document.body](https://github.com/bdougherty/BigScreen/blob/v2.0.4/src/bigscreen.js#L193) as target.